src/components/Pages/ManageWiki/Features/EntityMapping.vue

```diff
@@ -24,20 +24,43 @@
         Some tools assume items with special meanings under certain identifiers. 
 		Here, you can map a item on your Wikibase instance to an item on Wikidata.
       </v-card-text>
-	  <ItemsTable/>
+	  <ItemsTable ref="itemTable" @save="onSave" />
 	</v-card>
 </div>
 </template>
 
 <script>
-import PropertiesTable from './PropertiesTable.vue';
-import ItemsTable from './ItemsTable.vue';
+import PropertiesTable from './PropertiesTable.vue'
+import ItemsTable from './ItemsTable.vue'
 
 export default {
   name: 'EntityMapping',
   components: { PropertiesTable, ItemsTable },
-  props: [],
+  props: ['wikiId'],
   methods: {
+    toMapping (entities) {
+      const mappings = []
+      for (var i = 0; i < entities.length; i++) {
+        const mapping = entities[i]
+        mappings[mapping.local] = mapping.wikidata
```

Comment:
I think it should be the opposite `mappings[mapping.wikidata] = mapping.local` because the tools that read the setting should be parsing it by wikidata's IDs.

Reply:
done!